app/models/site_history.rb

```diff
@@ -1,7 +1,33 @@
 # frozen_string_literal: true
 
 class SiteHistory < ApplicationRecord
+  include DateValidateable
+
   belongs_to :planning_application
 
-  validates :date, :application_number, :description, :decision, presence: true
+  alias_attribute :reference, :application_number
+
+  normalizes :decision, with: ->(decision) { normalize_decision(decision) }
```

Comment:
Not sure about this - might need to do a distinct query on the PAAPI data to see the full extent of the variations